internal/remoteexec/command.go

```diff
@@ -62,7 +63,7 @@ func CommandOnPodByNames(c kubernetes.Interface, podNamespace, podName, cntName
 		return nil, nil, err
 	}
 
-	err = exec.Stream(remotecommand.StreamOptions{
+	err = exec.StreamWithContext(context.Background(), remotecommand.StreamOptions{
```

Comment:
this should be exposed at function level (add context at first argument) and the context should be instantiated at top-level rather than created here